app/views/items/correspondence_catalog.html.erb

```diff
@@ -0,0 +1,25 @@
+<%# overridden from orchid to add assets %>
+
+<% @ext_css = [ "datatables.min" ] %>
+<% @ext_js = [ "datatables.min", "corresp_catalog_datatables" ] %>
+
+<%# for instructions about how to use the search preset feature, please refer
+to docs/search_preset.md %>
+
+<%# provide the following:
+  - @res : API response object ($api.query(options))
+  - @route_path : string representing route used to call this template ("image_gallery_path")
+  - @content_label : OPTIONAL overrides @title for h1
+  - @preset_class : OPTIONAL adds CSS class on <div>
+%>
+
```

Comment:
I think these comments (after "provide the following") should be in the controller not the view, since the variables are set in the controller. Or else take it out entirely, I think the link to search_preset is enough